config/src/tests/configgen/vector_inserter.cpp

```diff
@@ -106,7 +107,22 @@ TEST("require that vector of string can be inserted") {
     root.addString("foo");
     root.addString("bar");
     root.addString("baz");
-    VectorInserter<vespalib::string> inserter(vector);
+    VectorInserter inserter(vector);
+    root.traverse(inserter);
+    ASSERT_EQUAL(3u, vector.size());
+    ASSERT_EQUAL("foo", vector[0]);
+    ASSERT_EQUAL("bar", vector[1]);
+    ASSERT_EQUAL("baz", vector[2]);
+}
+
+TEST("require that vector of string can be inserted") {
```

Comment:
Consider renaming (and possibly factoring out) test since it's currently duplicated with the above test and only differs in vector type.

Reply:
Fixed